clusterloader2/pkg/config/template_provider.go

```diff
@@ -240,11 +241,24 @@ func LoadCL2Envs() (map[string]interface{}, error) {
 			return nil, goerrors.Errorf("unparsable string in os.Eviron(): %v", keyValue)
 		}
 		key, value := split[0], split[1]
-		mapping[key] = value
+		mapping[key] = unpackStringValue(value)
```

Comment:
Are you sure it won't break any existing test?

Reply:
I'm 99% sure as of now.
Currently, we only use this feature in two places: scheduler_throughput and access_tokens. Both are enabled in the presubmit so we'll know for 100% in ~30min